### PR TITLE
br: Delete useless code

### PR DESCRIPTION
**Pull request by joccau**
Signed-off-by: joccau <zak.zhao@pingcap.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/32131

Problem Summary:
Delete code about `cdclog` in `br`. 
It related to https://github.com/pingcap/ticdc/pull/3599

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

```release-note
None
```

### CODE REVIEW

**Review by glorv — APPROVED** [review]
LGTM

**Review by fengou1 — APPROVED** [review]

File: br/pkg/lightning/backend/local/iterator.go [hunk 1]
```diff
@@ -21,12 +21,39 @@ import (
 
 	"github.com/cockroachdb/pebble"
 	sst "github.com/pingcap/kvproto/pkg/import_sstpb"
-	"github.com/pingcap/tidb/br/pkg/kv"
+
```
Comment (hawkingrei) [inline br/pkg/lightning/backend/local/iterator.go:24]: Please remove this empty line.

**Pull request merged**